src/crimson/osd/pg.cc

```diff
@@ -507,7 +507,7 @@ void PG::on_active_actmap()
         logger().debug("{}: trimmed snap={}", *this, trimmed);
       });
     }
-  ).finally([this, pg_ref] {
+  ).finally([this, pg_ref=std::move(pg_ref)] {
```

Comment:
This avoids a refcount mutation, but shouldn't actually fix a bug, right?

Reply:
Right